book/09-git-and-other-scms/sections/client-hg.asc

```diff
@@ -1,19 +1,19 @@
-==== Git and Mercurial
+==== Git і Mercurial
 
 (((Interoperation with other VCSs, Mercurial)))
 (((Mercurial)))
-The DVCS universe is larger than just Git.
-In fact, there are many other systems in this space, each with their own angle on how to do distributed version control correctly.
-Apart from Git, the most popular is Mercurial, and the two are very similar in many respects.
+Всесвіт розподілених систем контролю версій значно більший, ніж просто Git.
+Насправді, існує багато інших систем, кожна зі своїм власним поглядом на процес розподіленого контролю версій.
+Окрім Git, найпопулярнішою з них є Mercurial, і ці дві системи мають багато спільного.
 
-The good news, if you prefer Git's client-side behavior but are working with a project whose source code is controlled with Mercurial, is that there's a way to use Git as a client for a Mercurial-hosted repository.
-Since the way Git talks to server repositories is through remotes, it should come as no surprise that this bridge is implemented as a remote helper.
-The project's name is git-remote-hg, and it can be found at https://github.com/felipec/git-remote-hg[].
+Хорошою новиною є те, що якщо ви надаєте перевагу Git, але вам доводиться працювати з проектами, код яких знаходиться в системі Mercurial, існує спосіб використання Git-клієнта для роботи з Mercurial.
```

Comment:
> для роботи з репозиторієм Mercurial

Може скластися хибна думка в читача шо меркуріал то репозиторій.

Може "з репозиторієм на Mercurial"
On Sep 19, 2015 09:33, "Kyrylo Yatsenko" <notifications@github.com> wrote:

> In book/09-git-and-other-scms/sections/client-hg.asc
> <https://github.com/progit/progit2-uk/pull/15#discussion_r39919645>:
>
> >
> > -The good news, if you prefer Git's client-side behavior but are working with a project whose source code is controlled with Mercurial, is that there's a way to use Git as a client for a Mercurial-hosted repository.
> > -Since the way Git talks to server repositories is through remotes, it should come as no surprise that this bridge is implemented as a remote helper.
> > -The project's name is git-remote-hg, and it can be found at https://github.com/felipec/git-remote-hg[].
> > +Хорошою новиною є те, що якщо ви надаєте перевагу Git, але вам доводиться працювати з проектами, код яких знаходиться в системі Mercurial, існує спосіб використання Git-клієнта для роботи з Mercurial.
>
> Я б усе ж таки наблизив до оригіналу: "Git-клієнта для роботи з Mercurial"
> -> "Git у якості клієнта для роботи з репозиторієм Mercurial".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/progit/progit2-uk/pull/15/files#r39919645>.
>